src/wallet/scriptpubkeyman.cpp

```diff
@@ -201,6 +201,11 @@ IsMineResult IsMineInner(const LegacyScriptPubKeyMan& keystore, const CScript& s
         }
         break;
     }
+    case TxoutType::TX_BARE_DEFAULT_CHECK_TEMPLATE_VERIFY_HASH:
+    {
+        ret = IsMineResult::NO;
```

Comment:
Is this a placeholder? Maybe this merits commentary about why we'd never consider a bare CTV hash as owned by a wallet (i.e. do we expect that only transactions using this output might be IsMine?).

Reply:
This is simply the no-behavior-change from without TX_BARE_DEFAULT_CHECK_TEMPLATE_VERIFY_HASH, since before TX_NONSTANDARD was set to IsMine::No, overridable from the watchonly.

The future logic will not be to mark things as IsMine, but to use the IsTrusted recrusive scanning https://github.com/bitcoin/bitcoin/pull/16766 with a patch to treat bare CTV as trusted (since the TXID must be stable)